.travis.yml

```diff
@@ -0,0 +1,67 @@
+sudo: required
+
+language: generic
+
+services:
+  - docker
+
+env:
+  global:
+    - qemu_version=4.0.0-2
+    - target_version=1.1.2
+  matrix:
+    - target_arch=amd64 qemu_arch=x86_64
+    - target_arch=armv7 qemu_arch=arm
+#    - target_arch=armv6 qemu_arch=arm <------ NOT SUPPORTED BY DEBIAN STRETCH BASE IMAGE
+    - target_arch=arm64 qemu_arch=aarch64
+
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
+  - mkdir $HOME/.docker
+  - 'echo "{" > $HOME/.docker/config.json'
+  - 'echo "  \"experimental\": \"enabled\"" >> $HOME/.docker/config.json'
+  - 'echo "}" >> $HOME/.docker/config.json'
+  - sudo service docker restart
+  - git clone https://github.com/sstephenson/bats.git
```

Comment:
Hello Everyone,

Out of curiosity. Is there a specific reason that the old bats was preferred and not the new repository (https://github.com/bats-core/bats-core). Apparently the sstephenson/bats is abandoned (https://github.com/sstephenson/bats/pull/248) and a new community has already emerge (https://github.com/sstephenson/bats/issues/150).

KR,

Vasileios